CHANGES.md

```diff
@@ -197,6 +197,7 @@ Released on January 5, 2022.
     method. [[#1680]]
  -  Added `StateStoreExtensions.EncodeKey()` static method.  [[#1689]]
  -  Added `StateStoreExtensions.DecodeKey()` static method.  [[#1689]]
+ -  Added `TrieStateStsore.CopyStates` method.  [[#1691]]
```

Comment:
```suggestion
 -  Added `TrieStateStsore.CopyStates()` method.  [[#1653], [#1691]]
```

FYI #1653 was already linked once before, so you don't have to add its target this time:

<https://github.com/planetarium/libplanet/blame/ae69731508cbedde682b1d0d47c4a25a00a3ed3e/CHANGES.md#L328>

Reply:
Thank you! There is also a typo and I'll fix it also.